docs/conf.py

```diff
@@ -85,31 +77,15 @@
 
```

Comment:
Line 51:
```
language = 'en'
```

Let's start getting rid of `WARNING: Invalid configuration value found: 'language = None'. Update your configuration to a valid language code. Falling back to 'en' (English).`